### DISCUSSION TITLE
Is the old Web.WebUrlFromPageUrlDirect from SharePointClient removed?

### DISCUSSION THREAD

**Discussion by planetpallavi** [pnp/pnpcore#1225] [Q&A]
I was looking at ways to reach a site/web directly by a document absoluteurl... 
Unless I do some string operations to retrieve the siteUrl, is there a way I can retrieve it?
There was a handy Web.WebUrlFromPageUrlDirect which I used previously... but cant see it now...

**Comment by jansenbe** [comment]
@planetpallavi :: are you referring to CSOM when you talk about the "old"?
